docs/views/examples/pass-data/index.html

```diff
@@ -80,9 +80,17 @@ <h3 class="govuk-heading-s">
       <pre class="app-code" tabindex="0"><code>&lt;input type="radio" name="claimant[over-18]" value="yes" {%raw%}{{ checked("['claimant']['over-18']", "yes") }}{%endraw%}&gt;</code></pre>
 
       <h3 class="govuk-heading-s">
-        Setting default data
+        Setting default values
       </h3>
 
+      <p>
+        Default values are pre-filled data in some of your form’s fields. For example, the day’s date on a form that records the date of submission.
```

Comment:
suggest: For example, prototyping a user returning to a form they have previously filled in